app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -460,6 +460,8 @@ protected void onNewIntent(Intent intent) {
         // This is called when the user press Home again while already browsing MainActivity
         // onResume() will be called right after, hiding the kissbar if any.
         // http://developer.android.com/reference/android/app/Activity.html#onNewIntent(android.content.Intent)
+        onBackPressed();
+        hideKeyboard(); // Doesn't actually seem to work right now, but if it every does lets leave it ere
```

Comment:
We discussed this, it will display the keyboard if your setting says "display keyboard on app open".

We can update the comment before merging